superset/assets/javascripts/explore/main.css

```diff
@@ -134,3 +138,7 @@
 .datasource-container {
   overflow: auto;
 }
+
+.edit-adhoc-metric-save-btn {
```

Comment:
can you use the existing `m-r-5` class? (I think this is the name)

Reply:
Oh cool! I didn't realize that existed